### PR TITLE
Testing the presence and formatting of the timestamp on the top story

### PR DESCRIPTION
**Pull request by jamesbrumpton**
**Overall change:** _Adding Cypress test to check the presence and formatting of the timestamp within the top story._

**Code changes:**

- _Adding Cypress test for the top story timestamp._
- _Test only runs against the top story._
- _Test collects unix data, applies `moment.tz`, locale and formats to match what is found on the page._
- _Test only runs on Thai._
- _Test has been added to `testsThatAlwaysRun`._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/pages/frontPage/tests.js [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+      it('should render a formatted timestamp in the top story', () => {
+        cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+          ({ body }) => {
+            const { language } = body.metadata;
+            const { timestamp } = body.content.groups[0].items[0];
+            const formattedTimestamp = moment
+              .tz(timestamp, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+            cy.get('section li')
```
Comment (ghost) [inline cypress/integration/pages/frontPage/tests.js]: This doesn't get the `topStory`. as top story is not in a list. It fails on local.
```suggestion
            cy.get('section')
```

Comment (jamesbrumpton) [reply cypress/integration/pages/frontPage/tests.js]: Fixed :)

**Review by ghost — APPROVED** [review]

**Review by oluoluoxenfree — APPROVED** [review]

File: cypress/integration/pages/frontPage/tests.js [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+            const { language } = body.metadata;
+            const { timestamp } = body.content.groups[0].items[0];
+            const formattedTimestamp = moment
+              .tz(timestamp, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+            cy.get('section')
+              .eq(0)
+              .within(() => {
+                cy.get('div div time').should('contain', formattedTimestamp);
```
Comment (dr3) [inline cypress/integration/pages/frontPage/tests.js]:
```suggestion
                cy.get('time').should('contain', formattedTimestamp);
```

**Review by dr3 — APPROVED** [review]

**Comment by jamesdonoh** [comment]
This is causing failures when run against the Thai front page such as:
```
  1) frontPage - thai - Canonical Running testsToAlwaysRun for thai frontPage should render a formatted timestamp in the top story:

     CypressError: Timed out retrying: expected '[ <time.StyledTimestamp-um718p-0.eKyewn>, 11 more... ]' to contain '10 กันยายน 2019'
```
Think this is intending to check the timestamp on the first top story but it's getting back all the timestamps for all the stories.

**Pull request merged**